INSTALL.rst

```diff
@@ -13,19 +53,58 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
     $ workon inspirehep
     (inspirehep)$ cdvirtualenv
     (inspirehep)$ mkdir src
-    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git inspirehep
+    (inspirehep)$ git clone https://github.com/inspirehep/inspire-next.git src/inspirehep
 
 
 Install requirements
 ====================
 
-Use `pip` to install all requirements:
+Use `pip` to install all requirements, it's recommended to upgrade pip and
+setuptools to latest too:
 
 .. code-block:: console
 
+    (inspirehep)$ pip install --upgrade pip setuptools
     (inspirehep)$ cdvirtualenv src/inspirehep
     (inspirehep)$ pip install -r requirements.txt --pre --exists-action i
```

Comment:
We can probably get rid of the `--exists-action i` now... it's an historical artifact of the Invenio/INSPIRE overlay concept.